v4/source/definitions.yaml

```diff
@@ -804,6 +804,18 @@ components:
         IconData:
           description: Base64 encoded svg image
           type: string
+    AutocompleteListItem:
+      type: object
+      properties:
+        Item: 
+          description: Predicted suggestion for a static list and dynamic arguments
+          type: string
+        Hint:
+          description: Hint about suggested input
```

Comment:
0/5 when I was working on enabling Jira, I was a little puzzled as to what should be included in the hint; maybe an example and/or a style guide would be useful?